src/libs/blueprint/conduit_blueprint_mesh.hpp

```diff
@@ -642,6 +650,11 @@ namespace topology
 //-----------------------------------------------------------------------------
 namespace matset
 {
+    //-------------------------------------------------------------------------
+    void CONDUIT_BLUEPRINT_API generate_strip(conduit::Node& fields,
```

Comment:
This should be a private helper b/c it really can't be used outside the context of other generate_strip methods.

Reply:
Further review: don't think we need this this guy?
(Not being called)

Sorry if I am commenting while things are evolving.